chains/evm/calls/transactor/monitored/monitored.go

```diff
@@ -25,6 +28,11 @@ type RawTx struct {
 	creationTime time.Time
 }
 
+type RawTxWithTraceID struct {
```

Comment:
```suggestion
type PendingTx struct {
```

I hate the naming of what it is + something

Reply:
PendingTx does not display the changes made. Moreover, I am not aware of any conventions for restricting using `with` in naming. I like how basically it displays how this struct different from basic one

Reply:
It doesn't need to reflect the changes. That's the point of wrapping it.
I think personally it is invalid as we don't name things with what is inside, but with what does the concept of it do. 
The concept of it is it is a pending transaction that we track until is finished. The trace part of it is irrelevant to what it does.

If it is named something + something it should be 2 different things from the start (that is my opinion of structures and functions).
One name means it is part of one conceptual thing.

Reply:
And then the naming of pending transaction map makes sense as it contains pending transactions.

Reply:
then it is more right just to make this field to be part of RawTx. I do not see how naming it Pending makes it easier to understand the changes.
I disagree about naming by the concept of what it does since what it does is an Interface, and state storing is a struct. Hence struct name should display it content not behaviour

Reply:
Ofcs there is always should a bit of common sense, so we would need to name something like `RawTxWithTraceIDWithContextWithGas` we will use it and name it somehow different or first of all understand why we are in such situation in first place.

Reply:
That's my point. 
This is the beginning of such a situation.
If we find ourselves we need another field in this struct which inevitably we will we would have to change the naming again.

My point is the naming should always be what that particular code is (conceptually) not what it contains and then you don't run into that problem.

If the name is `X`, that will always be X whatever the fields are in it.
That is the whole point of encapsulating a piece of information.